Library/Formula/compcert.rb

```diff
@@ -13,7 +13,7 @@ class Compcert < Formula
   end
 
   depends_on "ocaml" => :build
-  depends_on "coq" => :build
+  depends_on "homebrew/versions/coq84" => :build
```

Comment:
Can't have mandatory cross-tap dependencies, even `:build` ones. If it needs this version for building you'll need to vendor it into the `buildpath` and fix the `PATH` to find it.